.github/actions/prepare-ce-test-env/action.yml

```
@@ -56,16 +56,16 @@ runs:
         tarantool-version: '${{ inputs.tarantool-version }}'
 
     # Here is a workaround for Tarantool 3.0 installation until it is supported
-    # by setup-tarantool. This step will install the latest pre-released
-    # Tarantool 3.0 from repository.
+    # by setup-tarantool. It installs alpha3 version, because the latest
+    # versions fail some tests due to the issue: https://github.com/tarantool/tarantool/issues/9275
```

It seems to me that this will make it clearer what should be, but why not. Up to you.
```suggestion
    # by setup-tarantool. This step will install the latest pre-released
    # Tarantool 3.0 from repository.
    #
    # Now it installs alpha3 version, because the latest versions fail some tests due to
    # the issue: https://github.com/tarantool/tarantool/issues/9275
```

---

Done.